Chapters/04.02-probability.ipynb

```diff
@@ -1267,7 +1267,7 @@
     "\n",
     "Okay, so that explains part of the story. I've explained a little bit about how continuous probability distributions should be interpreted (i.e., area under the curve is the key thing), but I haven't actually explained what the `stats.norm.pdf()` function actually calculates. Equivalently, what does the formula for $p(x)$ that I described earlier actually mean? Obviously, $p(x)$ doesn't describe a probability, but what is it? The name for this quantity $p(x)$ is a **_probability density_**, and in terms of the plots we've been drawing, it corresponds to the *height* of the curve. The densities themselves aren't meaningful in and of themselves: but they're \"rigged\" to ensure that the *area* under the curve is always interpretable as genuine probabilities. To be honest, that's about as much as you really need to know for now. [^note5]\n",
     "\n",
-    "[^note5]: For those readers who know a little calculus, I'll give a slightly more precise explanation. In the same way that probabilities are non-negative numbers that must sum to 1, probability densities are non-negative numbers that must integrate to 1 (where the integral is taken across all possible values of $X$). To calculate the probability that $X$ falls between $a$ and $b$ we calculate the definite integral of the density function over the corresponding range, $\\int_a^b p(x) \\ dx$. If you don't remember or never learned calculus, don't worry about this. It's not needed for this book."
+    "[^note5]: For those readers who know a little calculus, I'll give a slightly more precise explanation. In the same way that probabilities are non-negative numbers that must sum to 1, probability densities are non-negative numbers that must integrate to 1 (where the integral is taken across all possible values of $X$). To calculate the probability that $X$ falls between $a$ and $b$ we calculate the definite integral of the density function over the corresponding range, $\\int_a^b p(x) \\ \\mathrm{d}x$. If you don't remember or never learned calculus, don't worry about this. It's not needed for this book."
```

Comment:
d is a constant and no variable, therefore it should not be italic (maybe pedantic).

Reply:
Excellent, duly noted!